disnake/ext/commands/params.py

```diff
@@ -680,18 +710,27 @@ def parse_annotation(self, annotation: Any, converter_mode: bool = False) -> boo
             self.min_value = annotation.min_value
             self.max_value = annotation.max_value
             annotation = annotation.underlying_type
+
+            if self.min_value < -(2**53) or self.max_value > 2**53:
+                self.large = True
```

Comment:
Wdym by "if something doesn't check out"?

Reply:
When `max_value > 2**53` but `large` is set to false, for example.
I suppose the implicit behavior is *fine* as well, I'd just like to avoid situations where someone sets a high `max_value` and wonders why they can enter letters in their slash command, since `large` just uses a string parameter under the hood.

[PEP 637](https://peps.python.org/pep-0637/) could've been nice, since it would've allowed something like `Range[5, 10, large=True]`, but that was rejected.
One possible solution for having users explicitly specify the behavior they want would be a `LargeRange` type; seems a bit clunky, but I can't think of a better solution right now (@onerandomusername ideas?).

Reply:
0fa5e2a